charts/gateway-helm/templates/envoy-proxy-topology-injector-webhook.yaml

```diff
@@ -28,10 +43,21 @@ webhooks:
         apiGroups: [""]
         apiVersions: ["v1"]
         resources: ["pods/binding"]
+    {{- if not $gatewayNamespaceMode }}
     namespaceSelector:
       matchExpressions:
         - key: kubernetes.io/metadata.name
           operator: In
           values:
             - {{ .Release.Namespace }}
+    {{- else if gt (len $watchedNamespaces) 0 }}
+    namespaceSelector:
```

Comment:
there is an issue with this approach, with multiple envoy-gateway running in 1 cluster, there's no way to decide which special web hook server to call